src/sgame/sg_bot_util.cpp

```diff
@@ -599,6 +599,8 @@ int BotGetDesiredBuy( gentity_t *self, weapon_t &weapon, upgrade_t upgrades[], s
 		upgrades[i] = UP_NONE;
 	}
 
+	auto buyArmors = [&]()
+	{
 	for ( auto const &armor : armors )
```

Comment:
indentation is off wherever you added these lambdas?

Reply:
I did it like that to avoid conflicts, I thought at the time there may be other versions of this function around, for example in betterai. Not sure.